node/node.go

```diff
@@ -61,7 +61,8 @@ func NewNode(genDoc *genesis.Genesis, conf *config.Config,
 	txPool := txpool.NewTxPool(conf.TxPool, messageCh)
 
 	// TODO implement dequeue for recent transaction
-	str, err := store.NewStore(conf.Store)
+	str, err := store.NewStore(conf.Store,
+		genDoc.Params().TransactionToLiveInterval, genDoc.Params().SortitionInterval)
```

Comment:
We can pass these parameters through the config as private member.
Similar code can be found here: https://github.com/pactus-project/pactus/blob/a032b7befaef748617c9d9594c7418f36144fda8/sync/config.go#L18

Reply:
@b00f 
At first, I agreed with you about this change, but you didn't consider that hardcoding takes away flexibility from both users and us?

Reply:
We are not going to hardcore anything. Instead of passing new agruments we can use private field in config. Then we get some advantages:
1- We can define parameters with more semantic names in config like: `TxCacheSize` or `SortitionSessCacheSize`
2- We don't need to change the method signature
3- We can assign values based on the some logic outside store, example from `params`